src/GroupResolverHandlerInterface.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Drupal\og;
+
+/**
+ * @file
+ * Contains \Drupal\og\GroupResolverHandlerInterface.
+ */
+
+/**
+ * Interface OgContextHandlerInterface.
+ *
+ * @package Drupal\og
+ */
+interface GroupResolverHandlerInterface {
```

Comment:
I think this would be much clearer if this was renamed to `GroupResolverInterface`, the 'handler' part is what makes it confusing to me, it brings up associations with `EntityHandlerInterface` :)